CONTRIBUTING.rst

```diff
@@ -42,35 +42,40 @@ To set up `lipyphilic` for local development:
     conda create -n lipyphilic-dev -c conda-forge python=3.10 pip
     conda activate lipyphilic-dev
 
-2. Fork `lipyphilic <https://github.com/p-j-smith/lipyphilic>`_
+2. Install and activate Rust::
+
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    source $HOME/.cargo/env
```

Comment:
probably worth linking [here](https://www.rust-lang.org/tools/install) instead in case someone is using Windows